sdk/samples/http_client/curl/src/azure_core_with_curl_bodyStream.cpp

```diff
@@ -98,8 +98,8 @@ void doGetRequest(Context context, HttpPipeline& pipeline)
 
   request.AddHeader("Host", "httpbin.org");
```

Comment:
This is an E2E test and not a sample.  I'd expect 0 samples for our http_client as we do not support general purpose usage of the http_client